summarize/src/diff.rs

```diff
@@ -10,8 +10,8 @@ pub struct DiffResults {
     pub total_time: SignedDuration,
 }
 
-fn build_query_lookup(query_data: &[QueryData]) -> HashMap<&str, usize> {
-    let mut lookup = HashMap::with_capacity(query_data.len());
+fn build_query_lookup(query_data: &[QueryData]) -> FxHashMap<&str, usize> {
+    let mut lookup = FxHashMap::default();
```

Comment:
`FxHashMap` is just a type alias for the standard library's `HashMap` so you should be able to call the same function like this `FxHashMap::with_capacity(query_data.len())`.

Reply:
Unfortunately that doesn't work because `with_capacity()` hardcodes the default `Hasher`. Instead we have to use the slight more verbose `FxHashMap::with_capacity_and_hasher(query_data.len(), Default::default())`.

Reply:
Thanks. Will fix it.